PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-Fixes # .
+**Issue that Pull Request is trying to fix(e.g. #IssueNumber):**
```

Comment:
I think it might be better if this said
```
Please fill in the issue number this pull request is fixing:

Fixes #
```
The reason is that GitHub looks for the `Fixes #` text to automatically close the linked issue. I don't think that will work without the specific text `Fixes` and then the issue number.

Reply:
I see. I added it back.